contrib/completion/labgrid-client.bash

```diff
@@ -0,0 +1,35 @@
+# complete labgrid-client
+_labgrid-client()
+{
+    local cur=${COMP_WORDS[COMP_CWORD]}
+    local prev=${COMP_WORDS[COMP_CWORD-1]}
+
+    case "$prev" in
+	-p)
+	    COMPREPLY=( $( compgen -W "$(labgrid-client complete places)" -- $cur ) )
+	    return 0
+	    ;;
+    esac
+
+    # completing an option
+    if [[ "$cur" == -* ]]; then
+        COMPREPLY=( $( compgen -W "-p -h -x" -- $cur ) )
+    else
```

Comment:
No need to split things up here. ``compgen`` will filter thing correctly depending on ``$cur``.